lib/game.rb

```diff
@@ -0,0 +1,71 @@
+class Game
+  attr_accessor :board, :player_1, :player_2
+
+  WIN_COMBINATIONS = [
+    [0, 1, 2],
+    [3, 4, 5],
+    [6, 7, 8],
+    [0, 4, 8],
+    [6, 4, 2],
+    [0, 3, 6],
+    [1, 4, 7],
+    [2, 5, 8]
+  ]
+
+  def initialize(player_1=Human.new("X"), player_2=Human.new("O"), board=Board.new)
+    @player_1 = player_1
+    @player_2 = player_2
+    @board = board
+  end
+
+  def current_player
+    self.board.turn_count % 2 == 0 ? self.player_1 : self.player_2
+  end
+
+  def over?
+    true if won? || draw?
+  end
+
+  def won?
+    x_cells = self.board.cells.each_index.select{|i| self.board.cells[i] == "X"}
+    o_cells = self.board.cells.each_index.select{|i| self.board.cells[i] == "O"}
+    WIN_COMBINATIONS.each do |combination|
+      return combination if combination.all? {|pos| x_cells.include?(pos)}
+      return combination if combination.all? {|pos| o_cells.include?(pos)}
+    end
+    false
+  end
+
+  def draw?
+    true if self.board.full? && !won?
+  end
+
+  def winner
+    self.board.cells[won?[0]] if won?
```

Comment:
Instead of calling `Game#won?` twice, consider using an if statement assignment. For example:

```ruby
if won = won?
  board.cells[won.first]
end
```

If `won?` returns true, then and only then will the local variable `won` be assigned, otherwise, the method will just abort with a false return value. But if `won` is assigned, we can use that variable instead of calling `won?` again within the if statement.